suse_migration_services/units/setup_host_network.py

```diff
@@ -88,6 +82,28 @@ def main():
         Command.run(
             ['systemctl', 'reload', 'network']
         )
+        sysconfig_network_providers = os.sep.join(
+            [root_path, 'etc', 'sysconfig', 'network', 'providers']
+        )
+        sysconfig_network_setup = os.sep.join(
+            [root_path, 'etc', 'sysconfig', 'network', '*']
+        )
```

Comment:
Probably cut and paste issue, re-assignment of `sysconfig_network_providers` and `sysconfig_network_setup`, not necessary.

Reply:
Thanks for catching this.  I removed it